demos/jupyterhub-pyspark-hdfs-anomaly-detection-taxi-data/load-test-data.yaml

```
@@ -8,13 +8,15 @@ spec:
     spec:
       containers:
         - name: load-ny-taxi-data
-          image: docker.stackable.tech/stackable/hadoop:3.3.4-stackable24.7.0
+          image: docker.stackable.tech/stackable/hadoop:3.4.0-stackable24.11.0
```

24.11 is not yet available. HDFS 3.4.0 image is available in SDP 24.7.0.

```suggestion
          image: docker.stackable.tech/stackable/hadoop:3.4.0-stackable24.7.0
```